.circleci/config.yml

```diff
@@ -155,13 +144,20 @@ commands:
       - run:
           name: Run e2e test
           command: |
-            echo "Running e2e test in a '<< parameters.namespace >>' namespace of '<< parameters.cluster-ip >>' in a '<< parameters.mode >>' mode"
+            echo "Running '<< parameters.mode >>' mode e2e test in a '<< parameters.namespace >>' namespace"
             cd demo
-            ./mvnw package
-            ./scripts/end-to-end_fruit_demo.sh << parameters.cluster-ip >> << parameters.namespace >> << parameters.mode >>
+
+            ./mvnw clean package
+
+            # Set a correct deploymentMode in a component configuration
+            sed -i -E "s/(deploymentMode:).*/\1 \"<< parameters.mode >>\"/g" fruit-backend-sb/target/classes/META-INF/dekorate/component.yml
```

Comment:
if you use the new API -> halkyon.io and snapshot build of dekorate, then the file name to be used is now `halkyon.yml` and not anymore `component.yml`